app/server/app/routes/api.js

```diff
@@ -14,14 +19,15 @@ const router = express.Router();
 const s3Bucket = process.env.S3_PUBLIC_BUCKET;
 const s3Region = process.env.S3_PUBLIC_REGION;
 
-const formioProjectUrl = process.env.FORMIO_PROJECT_URL;
-const formioFormId = process.env.FORMIO_FORM_ID;
-const formioApiKey = process.env.FORMIO_API_KEY;
-
-const formioHeaders = { headers: { "x-token": formioApiKey } };
-
 router.use(ensureAuthenticated);
 
+// --- verification used to check if user has access to the /helpdesk route
+router.get("/helpdesk-access", (req, res) => {
+  // TODO: return 200 if user is a member of the correct EPA WAA groups, else
+  // send an error status code that the client app's fetch will catch (401?)
+  res.sendStatus(200);
+});
+
```

Comment:
@dpgalloway this is intentionally in the server's `/routes/api` file. It needs the same `ensureAuthenticate` check (not an elevated check just for helpdesk users – more on that in a bit), as it's called to determine if a regular authorized user is also a helpdesk user.

Inside the callback is where you'll want to verify the user is in the correct EPA WAA groups. If they are, you can return a simple 200 status; if they're not, you should return an error status code (probably 401, but the client app doesn't actually care, as long as it's an HTTP error code – it just needs to be caught in the client app fetch's `catch` block and that'll indicate the user doesn't have access).